helm/prometheus-meta-operator/values.yaml

```diff
@@ -54,13 +54,24 @@ prometheus:
     ca: "ca"
     crt: "crt"
     key: "key"
+  evaluationInterval: "60s"
   heartbeat:
     opsgenieKey: ""
   host: "prometheus:9090"
   logLevel: info
   letsencrypt: false
   mayu: ""
 
+  retention:
+    ## Retention duration for prometheus.
+    duration: "2w"
+
+  ## Default scrape interval for prometheus jobs.
+  scrapeInterval: "60s"
+
+  ## Prometheus container image version.
+  # version: ""
```

Comment:
@hervenicol if we set prometheus.version here it will be used everywhere. I can add the repository also

Reply:
imageRepository added